notebooks/explainable-ai-1-basic/explainable-ai-1-basic.ipynb

```diff
@@ -0,0 +1,456 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    %pip install -q "openvino&gt;=2023.1.0" opencv-python tqdm
</pre><p>we prefer to use the latest version of openvino as basic point for notebook installation, please use "openvino&gt;=2024.2.0"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2150/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>